### DISCUSSION TITLE
can we change the redirect url when login with OAuth google from "to continue to randomecharacter.supabse.co" to "to continue to mysite.com"

### DISCUSSION THREAD

**Discussion by tle95** [supabase/supabase#26900] [Questions]
![image](https://github.com/supabase/supabase/assets/139814124/b1b193f2-5475-4142-8a94-97e9b72883fd)

This randome url will definitely scare new user when they try to sign in.

I know that we have a Pro plan for this, but I am not ready to upgrade fro Pro plan just for this feature. I will be ready to upgrade if i have some users and generate some revenue.

**Comment by GaryAustin1** [comment]
There use to be this method. 
https://github.com/orgs/supabase/discussions/2925
Involved actual communication with Google.
I do not know if it exists anymore or not.
